lib/src/models/data-item.dart

```diff
@@ -1,15 +1,14 @@
 import 'dart:convert';
 import 'dart:typed_data';
 
-import 'package:json_annotation/json_annotation.dart';
+import 'package:arweave/src/utils/bundle_tag_parser.dart';
 
 import '../crypto/crypto.dart';
 import '../utils.dart';
 import 'models.dart';
 
-part 'data-item.g.dart';
+final MIN_BINARY_SIZE = 1044;
```

Comment:
Remove if unused

Reply:
It's used. That's the min data item size.